scripts/ci/clang-format.sh

```diff
@@ -25,9 +25,10 @@ _clang_format() {
       if [ $INPLACE ]; then
         ${CLANG_FORMAT} -i ${FILE}
         echo "File ${FILE} updated"
+      else
```

Comment:
This will make the "all clear" message confusing. How about adding a new counter, say, `CHANGED` and update when a file is reformatted, and then have a third message at the end: "${CHANGED} files were reformatted."?

Reply:
Good point, I like that idea